docs/user-guide/sparkother.rst

```diff
@@ -0,0 +1,98 @@
+.. _data-curator-sparkother:
+
+==========================================================================
+Reading and writing datasets with NeMo Curator and Apache Spark
+==========================================================================
+-----------------------------------------
+Background
+-----------------------------------------
+
+NeMo Curator uses the ``DocumentDataset`` to read and write ``JSONL`` and ``Parquet`` files. 
+It is a wrapper around a `Dask DataFrame <https://docs.dask.org/en/stable/dataframe.html>`_.  Apache Spark can read and write JSONL and Parquet files generated by NeMo Curator, and similarly, NeMo Curator can work with the outputs generated by Spark.
+
+-----------------------------------------
+Usage
+-----------------------------------------
+
+To demonstrate how this would work, consider the following example:
+
+.. code-block:: python
+
+    import dask.dataframe as dd
+    import pandas as pd
+    from nemo_curator.datasets import DocumentDataset
+
+    # Create sample data 
+    data = {
+        "id": [1, 2, 3],
+        "text": [
+            "This is a tiny story.",
+            "Another tiny story appears here.",
+            "Yet another tiny story for you."
+        ]
+    }
+
+    # Convert to a pandas DataFrame first
+    df = pd.DataFrame(data)
+
+    # Convert pandas DataFrame to DocumentDataset
+    stories_ds = DocumentDataset(dd.from_pandas(df, npartitions=2))
+
+    # Write the dataset to JSONL files
+    stories_ds.to_json("tiny_stories/", write_to_filename=False)
+
+This will create two JSONL files in the directory ``tiny_stories/``:
+
+::
+
+    tiny_stories/
+        0.part
+        1.part
+
+Apache Spark can read these files using standard APIs. Assuming you already have a Spark session running, you can read files in the directory using:
```

Comment:
```suggestion
Apache Spark can read these files using standard APIs. Let's first create a Spark session called ``NeMoCuratorExample``, then we can read files in the directory using:
```

Nit, but it looks like you already include the code to start a Spark session below.